### PR TITLE
(FPTI) add logs for axo

### PR DESCRIPTION
**Pull request by siddy2181**
### Groups who should review (if applicable)

❤️ Thank you!

### CODE REVIEW

File: src/connect/component.jsx [hunk 1]
```diff
@@ -39,7 +40,14 @@ export const getConnectComponent = async (merchantProps) => {
...
     });
 
-    getLogger().error("load_axo_error", { err: stringifyError(error) });
+    getLogger()
+      .track({
+        [FPTI_KEY.CONTEXT_TYPE]: "CMID",
+        [FPTI_KEY.CONTEXT_ID]: cmid,
+        [FPTI_KEY.EVENT_NAME]: `ppcp_axo_failure`,
+      })
+      .error("load_axo_error", { err: stringifyError(error) })
```
Comment (cgdibble) [inline src/connect/component.jsx:49]: Do we typically do a different error for `sendCountMetric` and `FPTI`?

File: src/connect/component.jsx [hunk 2]
```diff
@@ -72,7 +86,14 @@ export const getConnectComponent = async (merchantProps) => {
       },
     });
 
-    getLogger().error("init_axo_error", { err: stringifyError(error) });
+    getLogger()
+      .track({
+        [FPTI_KEY.CONTEXT_TYPE]: "CMID",
+        [FPTI_KEY.CONTEXT_ID]: cmid,
+        [FPTI_KEY.EVENT_NAME]: `ppcp_axo_failure`,
```
Comment (cgdibble) [inline src/connect/component.jsx:93]:
```suggestion
        [FPTI_KEY.EVENT_NAME]: `ppcp_connect_failure`,
```

File: src/connect/component.jsx [hunk 3]
```diff
@@ -72,7 +86,14 @@ export const getConnectComponent = async (merchantProps) => {
...
+      })
+      .error("init_axo_error", { err: stringifyError(error) })
```
Comment (cgdibble) [inline src/connect/component.jsx:95]:
```suggestion
      .error("init_connect_error", { err: stringifyError(error) })
```

File: src/connect/component.jsx (hunk 2 above)
Comment (cgdibble) [reply src/connect/component.jsx:93]: We'll also need to add `page_session_id` to all of these as well: https://github.com/paypal/paypal-sdk-constants/blob/main/src/fpti.js#L10

**Review by jshawl — APPROVED** [review]
🚀 question for a slack convo in the future: how did you know what to log and when to log it? (I am about to do the same in another component)

**Pull request merged**